config/crd/bases/registry.devfile.io_devfileregistries.yaml

```diff
@@ -36,13 +44,63 @@ spec:
         spec:
           description: DevfileRegistrySpec defines the desired state of DevfileRegistry
           properties:
-            foo:
-              description: Foo is an example field of DevfileRegistry. Edit DevfileRegistry_types.go
-                to remove/update
+            devfileIndexImage:
+              description: Sets the container image containing devfile stacks to be
+                deployed on the Devfile Registry
               type: string
+            k8s:
+              description: DevfileRegistrySpecK8sOnly defines the desired state of
+                the kubernetes-only fields of the DevfileRegistry
+              properties:
+                ingressDomain:
+                  description: Ingress domain for a Kubernetes cluster. This MUST
+                    be explicitly specified on Kubernetes. There are no defaults
+                  type: string
+              type: object
+            ociRegistryImage:
+              description: Overrides the container image used for the OCI registry.
+                Recommended to leave blank and default to the image specified by the
+                operator.
+              type: string
+            storage:
+              description: DevfileRegistrySpecStorage defines the desired state of
+                the storage for the DevfileRegistry
+              properties:
+                enabled:
+                  description: Instructs the operator to deploy the DevfileRegistry
+                    with persistent storage Enabled by default. Disabling is only
+                    recommended for development or test.
+                  type: boolean
+                ociRegistryImage:
+                  description: Configures the size of the devfile registry's persistent
+                    volume, if enabled. Defaults to 1Gi.
+                  type: string
+              type: object
+            tls:
+              description: DevfileRegistrySpecTLS defines the desired state for TLS
+                in the DevfileRegistry
+              properties:
+                enabled:
+                  description: Instructs the operator to deploy the DevfileRegistry
+                    with TLS enabled. Enabled by default. Disabling is only recommended
+                    for development or test.
+                  type: boolean
+                ociRegistryImage:
+                  description: Name of an optional, pre-existing TLS secret to use
+                    for TLS termination on ingress/route resources.
+                  type: string
+              type: object
           type: object
         status:
           description: DevfileRegistryStatus defines the observed state of DevfileRegistry
+          properties:
+            url:
+              description: 'INSERT ADDITIONAL STATUS FIELD - define observed state
```

Comment:
Format:
1. why `INSERT...` are all caps?
2. add a period after cluster.
3. It's not clear from the description of what this field is really for, e.g. what to put it.

Reply:
Huh, interesting. This is the filler code that the operator-sdk generates. Wonder if it got lumped in when I generated the CRDs. Lemme go take a look. That shouldn't be there thoughl.

Reply:
Updated to review sample comment generated by `operator-sdk`